### PR TITLE
fix(quantic): Added tooltips for quantic feedback component

### PR DESCRIPTION
**Pull request by SimonMilord**
[SFINT-5142](https://coveord.atlassian.net/browse/SFINT-5142)

Added tooltips for the quantic Feedback component. I used the labels `thisAnswerWasHelpful` & `thisAnswerWasNotHelpful`

DEMO:

DISLIKE TOOLTIP:

Screenshot 2023-08-25 at 11 03 22 AM

LIKE TOOLTIP:

Screenshot 2023-08-25 at 11 03 11 AM

DEMO IN smart snippet:

If you set like-label property to something like patate, the tooltip will be the same as label

image

DEMO IN Generated answer:

image

[SFINT-5142]: https://coveord.atlassian.net/browse/SFINT-5142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html
Comment (mmitiche) [inline packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html:1]: The labels used in the QuanticFeedback component can be specified using the properties `likeLabel`  and `dislikeLabel` , hence the need to use these properties as the title of the two buttons respectively.
Exactly like we are currently doing with the `aria-label` attribute.

File: packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html [hunk 1]
```diff
@@ -2,15 +2,15 @@
   <div class="slds-grid feedback__main-section">
     <span class="slds-var-p-right_medium slds-text-color_weak feedback__question"
       id="feedback__question">{question}</span>
-    <button aria-label={likeLabel} data-cy="feedback__like-button" aria-describedby="feedback__question" onclick={handleLike}
+    <button aria-label={likeLabel} data-cy="feedback__like-button" title={labels.thisAnswerWasHelpful} aria-describedby="feedback__question" onclick={handleLike}
```
Comment (mmitiche) [inline packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html:5]:
```suggestion
    <button aria-label={likeLabel} data-cy="feedback__like-button" title={likeLabel} aria-describedby="feedback__question" onclick={handleLike}
```

File: packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
       class={likeButtonClass}>
       <lightning-icon aria-hidden="true" class={likeIconClass} icon-name={likeIconName} size={size}
         variant={likeIconVariant}></lightning-icon>
       <template if:false={hideLabels}>
         <span class="feedback__button-label slds-var-m-left_xx-small">{likeLabel}</span>
       </template>
     </button>
-    <button aria-label={dislikeLabel} data-cy="feedback__dislike-button" aria-describedby="feedback__question" onclick={handleDislike}
+    <button aria-label={dislikeLabel} data-cy="feedback__dislike-button" title={labels.thisAnswerWasNotHelpful} aria-describedby="feedback__question" onclick={handleDislike}
```
Comment (mmitiche) [inline packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html:13]:
```suggestion
    <button aria-label={dislikeLabel} data-cy="feedback__dislike-button" title={dislikeLabel} aria-describedby="feedback__question" onclick={handleDislike}
```

File: packages/quantic/force-app/main/default/lwc/quanticFeedback/__tests__/quanticFeedback.test.js
Comment (mmitiche) [inline packages/quantic/force-app/main/default/lwc/quanticFeedback/__tests__/quanticFeedback.test.js:1]: The unit tests should be updated according to the comments I left bellow.

**Review by mmitiche — COMMENTED** [review]
Added some comments that need to be applied before merging, I would also suggest to test the components  when being used by the other Quantic component, namely the QuanticGeneratedAnswer component and the QuanticSmartSnipper component.
A demo with these two last components could be then added to the PR description to show that the title is set properly in both cases.

File: packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html (hunk 2 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html:13]: You have to change the `title={dislikeLabel}` just like Mehdi proposed it, and it's the parent component in the quanticGenAI that should be setting the label correctly.

File: packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html (hunk 1 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html:5]: See how the QuanticGeneratedAnswer component already sets the label correctly

Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticFeedback/quanticFeedback.html:5]: image

**Review by erocheleau — CHANGES_REQUESTED** [review]
Remove the labels you've added in the QuanticFeedback component, it's enough just to add the `title={likeLabel}`

**Review by erocheleau — APPROVED** [review]
👍

**Review by lvu285 — APPROVED** [review]

**Pull request merged**